src/Microphysics2M.jl

```diff
@@ -317,7 +317,12 @@ function autoconversion(
 ) where {FT <: Real}
 
     if q_liq < eps(FT)
-        return FT(0)
+        return (
```

Comment:
```suggestion
        return (;
```
Is this supposed to be a `NamedTuple`?

Reply:
(there are other similar returns in this PR where it seems like they're supposed to be `NamedTuples`)

Reply:
@charleskawczynski Thanks for reviewing this PR.
Yes, they are supposed to be named tupple. I thought by giving the outputs names I automatically get named tupples. For example: `c=(a=1, b=2)` and `a=1 b=2 d=(;a, b)` are both named tupples. Is the latter the preferred way?